test/adapters/parsers/json.spec.js

```diff
@@ -56,6 +57,27 @@ describe('parsers/json', function() {
         });
     });
 
+    it('can parse a file with nested arrrays correctly', function() {
```

Comment:
I also think there should be a test case for an array of objects with nested objects / arrays.